electives/82208.md

```diff
@@ -0,0 +1,29 @@
+# 82-208/79-456 Topics in European Studies: Eastern Europe: Society and Culture
+
+| Category |Difficulty  |
+| :------: | :--------: |
+| Reading Assignments | N/A |
+| Essays | N/A | 
+| Exams | None | 
+| Prerequisites | N/A | 
+  
+
+This course studies the complex societies and vibrant cultures of Eastern Europe from the Mongol invasion of Europe in the thirteenth century to the end of the Cold War in the late twentieth century. Some topics discussed in this course include nationalism, imperialism, Fascism, globalism, and separatism. The course aims to provide students with a better understanding of the historical processes that led to today’s socio-economic and political landscape of Eastern Europe. No prior knowledge of Slavic languages or Eastern European history is required.
+
+  
+
+## Lectures
+
+During lectures, students view documentaries and movies in addition to traditional lectures delivered using slide decks. In Spring 2022, Professor Kats also scheduled tours at the Carnegie Museum of Art and the Nationality Rooms in the Cathedral of Learning for students to gain a first-hand appreciation of Eastern European art and culture. Sometimes there is discussion about the reading assignments, but it is not a main emphasis of the lectures.
+
+  
+
+## Readings
+
+In Spring 2022, students need to read four books: The Bridge On the Drina by Ivo Andric, Journals by Mihail Sebastian, The Book of Laughter and Forgetting by Milan Kundera and Drive Your Plow Over the Bones of the Dead by Olga Tokarczuk. The books are well-selected and represent different historical periods and geographical regions of Eastern Europe. Two of the writers, Ivo Andric from Yugoslavia and Olga Tokarczuk from Poland are Nobel Prize in Literature laureates. Reading these books is enjoyable and should not be very time consuming (~6 hours per week).
```

Comment:
Can you format the 4 books to be more easily readable? An enumerated list would be good, like 

1. Book one
2. Book two
3. Book three